Library/brew.rb

```diff
@@ -50,6 +50,20 @@
   EOABORT
 end
 
+if OS.mac? && MacOS.version == "10.11" && MacOS::Xcode.version < "7.0"
```

Comment:
Reverted.

If we want to issue a sterner warning I'm happy to use the existing Doctor methods, but attempt to make them more fatal on 10.11 (i.e. you cannot try and install xyz until your Xcode/CLT is up-to-date), but the consensus seems to lean towards seeing how much the basic support reduces the level of Issues before we go more drastic.

If that's the case, this should be good to :rocket: once it's green on the bot again. I'll let one of you pull the trigger on this if you're happy.